hokusai/lib/default_branch.py

```diff
@@ -0,0 +1,9 @@
+from hokusai.lib.common import shout
+
+def detect_branch():
```

Comment:
Update: could you add just a bit of documentation here, for future reference? Something like "This function detects the default main/master branch from the repo and returns it as string" or something like that.